docs/catalog/cloud/choose-sustainable-region.md

```diff
@@ -0,0 +1,43 @@
+---
+version: 1.0
+submitted_by: markus-ntt-seidl
+published_date: TBD
+category: cloud
+tags: 
+ - cloud
+ - role:cloud-engineer
+ - size:medium
+---
+
+# Choose Sustainable Region for Workloads
+
+## Description
+
+Applications can have background tasks or jobs that run independent to the rest of the application. These workloads can be put into a region that is the most sustainable region of the used cloud provider.
+
+## Solution
+
+Run the workloads in a region that the used cloud provider labels as most sustainable. Choose the region that is near your application so that data transfer to and from the sustainable region minimal.
```

Comment:
I do think these two patterns describe different facts and details, can you check again?

Reply:
Had another look. So firstly, I think it's best if we separate this pattern into two patterns so we are following the 1:1 mapping. 

Secondly, if we were to separate the patterns: the other pattern's solution is _Choose a region that is the closest to the majority of where the network packets are going._ is fairly similar to _Choose the region that is still near to your application so that data transfer to and from the sustainable region is minimal._ Thoughts??